tests/fields/fields.py

```diff
@@ -932,6 +938,7 @@ class BlogPost(Document):
             tags = ListField(StringField())
             authors = ListField(ReferenceField(User))
             generic = ListField(GenericReferenceField())
+            access_list = ListField(required=False, default=[], choices=AccessLevelChoices, display_sep=u',')
```

Comment:
thanks for your advise, the u'' style string is for python2 I'm working on.I think its ok to set the default seperator to ''.
other things are all accepted,3x.